account_move_line_stock_info/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Move Line Stock Move",
+    "version": "8.0.1.0.0",
+    "depends": [
+        "stock_account"
+    ],
+    "author": "Eficent,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "installable": True,
+    "license": "AGPL-3",
+    "data": [
+        'security/ir.model.access.csv',
+        'views/account_move_line_view.xml',
+        'views/stock_move_view.xml',
+    ],
+    "images": [],
```

Comment:
Adding the stock move to the account move line is needed to ensure traceability. It's not necessary to make this field super visible in the journal entries/items.

Another module should implement an "Inventory Audit Report', such as https://help.sap.com/saphelp_sbo882/helpdata/en/45/22e90a73c80108e10000000a114a6b/content.htm